tokenizers/src/models/mod.rs

```diff
@@ -121,6 +121,7 @@ impl Model for ModelWrapper {
     }
 }
 
+#[derive(Serialize, Deserialize)]
```

Comment:
just to confirm, `TrainerWrapper` serde is using [external tagging](https://serde.rs/enum-representations.html#externally-tagged) (not internal) so that the serializations look like:
```json
{"BpeTrainer": { ... }
```

rather than
```json
{"type": "BpeTrainer", ... }
```
right?

Reply:
Very good remark.

I wasn't even sure, so I added a Python test to make sure we commit to the serialized format (or at least know when we break).

And yes it is.